google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -91,12 +92,13 @@ auto MakeChild(
 }
 
 Spans MakeBatchSinkSpans(Spans message_spans) {
-  int64_t max_otel_links = 128;
+  size_t max_otel_links =
```

Comment:
We spell `std::size_t`, or `auto const` unless you need to silence warnings about signed/unsigned comparisons.